src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -564,12 +383,6 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
       .expectStatus().isOk
       .expectBody()
       .jsonPath("$.draftAdjudications[0].id").isEqualTo(draftAdjudicationResponse.draftAdjudication.id)
-      .jsonPath("$.draftAdjudications[0].prisonerNumber").isEqualTo(testAdjudication.prisonerNumber)
-      .jsonPath("$.draftAdjudications[0].incidentDetails.dateTimeOfIncident")
-      .isEqualTo(testAdjudication.dateTimeOfIncidentISOString)
-      .jsonPath("$.draftAdjudications[0].incidentDetails.handoverDeadline")
-      .isEqualTo(testAdjudication.handoverDeadlineISOString)
-      .jsonPath("$.draftAdjudications[0].incidentDetails.locationId").isEqualTo(testAdjudication.locationId)
   }
 
   @Test
```

Comment:
Given the above changes, we can delete this `mark the incident statement as being complete` test